src/commonMain/kotlin/exchange.dydx.abacus/calculator/V2/TradeInput/TradeInputMarginModeCalculator.kt

```diff
@@ -19,6 +20,16 @@ internal class TradeInputMarginModeCalculator {
             marketId = tradeInput.marketId,
             subaccountNumber = subaccountNumber,
         )
+        val market = markets?.get(tradeInput.marketId)
+        val imf = market?.perpetualMarket?.configs?.initialMarginFraction ?: Numeric.double.ZERO
```

Comment:
Nit: Since this logic is used elsewhere, we can make it a computed property of `MarketConfigs`... Something like this:

```
data class MarketConfigs(...) {
 ....
 
     internal val maxMarketLeverage: Double
        get() {
            val imf = initialMarginFraction ?: Numeric.double.ZERO
            val effectiveImf = effectiveInitialMarginFraction ?: Numeric.double.ZERO
            return if (effectiveImf > Numeric.double.ZERO) {
                Numeric.double.ONE / effectiveImf
            } else if (imf > Numeric.double.ZERO) {
                Numeric.double.ONE / imf
            } else {
                Numeric.double.ONE
            }
        }
}
```